app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala

```diff
@@ -236,7 +241,7 @@ class StartController @Inject()(
     val newSessionData =
       request.sessionData.journeyStatus match {
         case Some(d: SubscriptionStatus.DeterminingIfOrganisationIsTrust) => d
-        case _                                                            => SubscriptionStatus.DeterminingIfOrganisationIsTrust(ggCredId, None, None)
+        case _                                                            => SubscriptionStatus.DeterminingIfOrganisationIsTrust(ggCredId, ggEmail, None, None)
       }
```

Comment:
why do we need to thread in the gg email now?

Reply:
this was related to the refactoring of the `StartController` I did to condense all the cases for `SubscriptionMissingData` into one case. In order to make that work I had to make sure that the gg email was always available from the session object.